mrp_workcenter_account_move/tests/__init__.py

```diff
@@ -0,0 +1,8 @@
+# coding: utf-8
```

Comment:
The new standard accepts: `# coding: utf-8` or `# -*- coding: utf-8 -*-` to avoid red pylint in CI-Travis.